lib/t/calls.ex

```diff
@@ -348,19 +392,35 @@ defmodule T.Calls do
     :ok
   end
 
-  defp in_match?(user_id, match_id) do
+  defp mate_match?(user_id, match_id) do
     Match
     |> where(id: ^match_id)
     |> where([m], m.user_id_1 == ^user_id or m.user_id_2 == ^user_id)
-    |> Repo.exists?()
+    |> select([m], {m.user_id_1, m.user_id_2})
+    |> Repo.one()
+    |> case do
+      {uid1, uid2} ->
+        [mate_id] = [uid1, uid2] -- [user_id]
+        mate_id
+
+      nil ->
+        nil
+    end
   end
 
-  @spec ensure_voicemail_allowed(uuid, uuid) :: :ok | {:error, reason :: String.t()}
+  @spec ensure_voicemail_allowed(uuid, uuid) :: {:ok, uuid} | {:error, reason :: String.t()}
```

Comment:
should be renamed if mate is returned

Reply:
it can be a step in a Multi

```elixir
Multi.new()
|> Multi.run(:mate, fn _repo, _changes ->
  mate = 
    Match
    |> ...
    |> Repo.one()

  if mate do
    {:ok, mate}
  else 
    {:error, :not_found}
   end
end)
```

Reply:
done in https://github.com/getsince/test3/pull/494